<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(254,253,0);
    font-size: 30px;
    border: 4px solid rgb(135,209,199);
    border-radius: 35px;
    border-style: outset;
    background-color: black;
    }

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}</string>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(254,253,0);
    font-size: 30px;
    border: 4px solid rgb(135,209,199);
    border-radius: 35px;
    border-style: outset;
    background-color: black;
    }

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}</string>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(254,253,0);
    font-size: 30px;
    border: 4px solid rgb(135,209,199);
    border-radius: 35px;
    border-style: outset;
    background-color: black;
    }

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}</string>
    </property>
    <property name="text">
     <string>✔️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_power">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(254,253,0);
    border: 4px solid rgb(135,209,199);
    border-radius: 35px;
    border-style: outset;
    background-color: black;
    }

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}</string>
    </property>
    <property name="text">
     <string>POWER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_powerLight">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_level">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	color: white; 
	background-color:transparent;
	font: 17pt ;
	}
QListView::Flow{
	setflow:LeftToRight;
}
QListWidget::item:selected {
	background: transparent;
	color: white;
    }
</string>
    </property>
    <item>
     <property name="text">
      <string>▮8▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮7▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮6▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮5▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮4▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮3▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮2▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮1▮</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_tdcs">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>61</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	color: white; 
	background-color:transparent;
	font: 17pt ;
	}
QListWidget::item:selected {
	background: transparent;
	color: white;
    }</string>
    </property>
    <item>
     <property name="text">
      <string>▮8▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮7▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮6▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮5▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮4▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮3▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮2▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮1▮</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_tdcs">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 14px;
font-size: 25px;
color: white;
   </string>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_time">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	color: white; 
	background-color:transparent;
	font: 17pt ;
	}
QListView::Flow{
	setflow:LeftToRight;
}
QListWidget::item:selected {
	background: transparent;
	color: white;
    }
,,</string>
    </property>
    <item>
     <property name="text">
      <string>20  </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45  </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>⊕   </string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_frequency">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	color: white; 
	background-color:transparent;
	font: 17pt ;
	}
QListView::Flow{
	setflow:LeftToRight;
}
QListWidget::item:selected {
	background: transparent;
	color: white;
    }
</string>
    </property>
    <item>
     <property name="text">
      <string>sΔ   </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>θ    </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>α    </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>β2   </string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: transparent;
}
QFrame { 
   border: 6px solid white;
} </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/image/res/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="battery">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
     color:black;
 }

 QProgressBar::chunk {
     background-color: green;
 }</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_cl">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
background-color:black;
font: 15pt ;</string>
    </property>
    <item>
     <property name="text">
      <string>▮1▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮2▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮3▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮4▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮5▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮6▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮7▮</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>▮8▮</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>461</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>connection level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>Session: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>Time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>ear clips connection:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ecc">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
background-color:black;
font: 12pt ;</string>
    </property>
    <item>
     <property name="text">
      <string>both connected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>left connected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>right connected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>not connected</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_change">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>change battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>battery level: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_btylevel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_endtime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_session">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_L">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>🗦 L 🗧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_R">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>🗦 R 🗧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>Intensity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_itsy">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>output:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>input: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_opt">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
background-color:black;
font: 12pt ;</string>
    </property>
    <item>
     <property name="text">
      <string>connected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>unconnected</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_ipt">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
background-color:black;
font: 12pt ;</string>
    </property>
    <item>
     <property name="text">
      <string>connected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>unconnected</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
	background-color:transparent;
	font: 17pt ;</string>
    </property>
    <property name="text">
     <string>user design time:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_h">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>620</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_m">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>620</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_s">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>620</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: transparent;
}
QFrame { 
   border: 6px solid white;
} </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: transparent;
}
QFrame { 
   border: 6px solid white;
} </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_history">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_record">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>600</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color:white;
   border: 2px solid white;
}

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}
</string>
    </property>
    <property name="text">
     <string>save record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_record">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>600</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color:white;
   border: 2px solid white;
}

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}
</string>
    </property>
    <property name="text">
     <string>start this session</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_battery">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 2px solid white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_changeLevel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color:white;
   border: 2px solid white;
}

QPushButton:pressed {
    border-style: inset;
    background-color: RGB(20,20,20);
}
</string>
    </property>
    <property name="text">
     <string>change battery level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_btylow">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>battery low please change battery</string>
    </property>
   </widget>
   <zorder>label_frame</zorder>
   <zorder>pushButton_up</zorder>
   <zorder>pushButton_down</zorder>
   <zorder>pushButton_check</zorder>
   <zorder>pushButton_power</zorder>
   <zorder>label_powerLight</zorder>
   <zorder>listWidget_level</zorder>
   <zorder>listWidget_tdcs</zorder>
   <zorder>label_tdcs</zorder>
   <zorder>listWidget_time</zorder>
   <zorder>listWidget_frequency</zorder>
   <zorder>label_logo</zorder>
   <zorder>battery</zorder>
   <zorder>comboBox_cl</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>comboBox_ecc</zorder>
   <zorder>pushButton_change</zorder>
   <zorder>label_5</zorder>
   <zorder>label_btylevel</zorder>
   <zorder>label_endtime</zorder>
   <zorder>label_session</zorder>
   <zorder>label_L</zorder>
   <zorder>label_R</zorder>
   <zorder>label_6</zorder>
   <zorder>label_itsy</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>comboBox_opt</zorder>
   <zorder>comboBox_ipt</zorder>
   <zorder>label_9</zorder>
   <zorder>spinBox_h</zorder>
   <zorder>spinBox_m</zorder>
   <zorder>spinBox_s</zorder>
   <zorder>label_frame_2</zorder>
   <zorder>label_frame_3</zorder>
   <zorder>listWidget_history</zorder>
   <zorder>listWidget_record</zorder>
   <zorder>pushButton_save</zorder>
   <zorder>pushButton_record</zorder>
   <zorder>spinBox_battery</zorder>
   <zorder>pushButton_changeLevel</zorder>
   <zorder>label_btylow</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1253</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
